packages/lib/src/components/internal/SecuredFields/lib/CSF/extensions/createSecuredFields.ts

```diff
@@ -245,13 +245,27 @@ export function setupSecuredField(pItem: HTMLElement, cvcPolicy?: CVCPolicyType,
             })
             .onTouchstart((pFeedbackObj: SFFeedbackObj): void => {
                 // re. Disabling arrow keys in iOS - need to disable all other fields in the form
-                if (this.config.disableIOSArrowKeys) {
-                    this.callbacks.onTouchstartIOS({ fieldType: pFeedbackObj.fieldType });
+                if (this.config.shouldDisableIOSArrowKeys) {
+                    /**
+                     * re. this.hasGenuineTouchEvents...
+                     *  There seems to be an issue with Responsive Design mode in Safari that means it allows setting focus on cross-origin iframes,
+                     *  without enabling the touch events that allow the "disableIOSArrowKeys" workaround to fully function.
+                     *  This results in a click on an securedFields *label* leading to, for example, the holderName field being disabled, but w/o access
+                     *  to the touch events that would let it re-enable itself.
+                     *
+                     *  So we prevent the "disableIOSArrowKeys" workaround unless we genuinely have touch events available.
+                     */
+                    if (this.hasGenuineTouchEvents || pFeedbackObj.hasGenuineTouchEvents) {
```

Comment:
What's the difference between `hasGenuineTouchEvents` in `this` vs in `pFeedbackObj`?

Reply:
The difference is where they've originated. `hasGenuineTouchEvents` here, has originated from a _touch_ in checkoutWeb e.g. on a `label` element whereas `pFeedbackObj.hasGenuineTouchEvents` has originated from a _touch_ directly into an input in the securedFields iframe. We don't know exactly where the "confirmation" that we have touch events is coming from, therefore we have to check for both.